Modules/Scripted/DICOMLib/Widgets/qSlicerDICOMExportDialog.cxx

```diff
@@ -289,24 +289,16 @@ void qSlicerDICOMExportDialog::examineSelectedItem()
         }
       exportable->setParent(this); // Take ownership to prevent destruction
       QString plugin = exportable->pluginClass();
-      if (!exportablesByPlugin.contains(plugin))
-        {
-        QList<qSlicerDICOMExportable*> firstExportableForPlugin;
-        firstExportableForPlugin.append(exportable);
-        exportablesByPlugin[plugin] = firstExportableForPlugin;
-        }
-      else
-        {
-        exportablesByPlugin[plugin].append(exportable);
-        }
+      exportablesByPlugin.insert(plugin, exportable);
     }
   }
   // Map the grouped exportables by confidence values so that the highest confidence is on top
-  QMap<double,QList<qSlicerDICOMExportable*> > exportablesByConfidence;
-  foreach(QList<qSlicerDICOMExportable*> exportablesForPlugin, exportablesByPlugin)
+  QMultiMap<double,QList<qSlicerDICOMExportable*> > exportablesByConfidence;
```

Comment:
When using `QMultiMap`, the values associated with a given key are managed using a list. For this reason, I think using `QMultiMap<double,qSlicerDICOMExportable*> exportablesByConfidence;` should be enough ?